hack/setup-test-cluster.sh

```diff
@@ -13,6 +13,20 @@ do
     ${OC_TOOL} label $node node-role.kubernetes.io/worker-cnf=""
 done
 
+# Label First master as grandmaster
+echo "[INFO]: Labeling first master as the ptp grandmaster"
+nodes=$(${OC_TOOL} get nodes --selector='node-role.kubernetes.io/master' -o name | sed -n 1p)
+for node in $nodes
+do
+    ${OC_TOOL} label $node ptp/grandmaster=""
+done
+
+echo "[INFO]: Labeling all the other hosts as ptp slaves"
+nodes=$(${OC_TOOL} get nodes -o name | sed 1d)
```

Comment:
If you do like this, you will skip the first result which is not necessarily the master you labelled above. Is that fine?

Reply:
I make a change in the first loop can you take a look?